mii/deployment.py

```diff
@@ -102,29 +64,36 @@ def deploy(task,
             assert (mii_config.torch_dtype() == torch.float), "MII Config Error: MII dtype and ZeRO dtype must match"
     assert not (enable_deepspeed and enable_zero), "MII Config Error: DeepSpeed and ZeRO cannot both be enabled, select only one"
 
-    task = mii.get_task(task)
-    mii.check_if_task_and_model_is_valid(task, model)
+    task = mii.utils.get_task(task)
+    mii.utils.check_if_task_and_model_is_valid(task, model)
     if enable_deepspeed:
-        mii.check_if_task_and_model_is_supported(task, model)
+        mii.utils.check_if_task_and_model_is_supported(task, model)
 
     logger.info(f"*************DeepSpeed Optimizations: {enable_deepspeed}*************")
 
-    create_score_file(deployment_name,
-                      task,
-                      model,
-                      enable_deepspeed,
-                      enable_zero,
-                      ds_config,
-                      mii_config)
+    # In local deployments use default path if no model path set
+    if model_path is None and deployment_type == DeploymentType.LOCAL:
```

Comment:
Do we need a similar if statement for `model_path is None and deployment_type == DeploymentType.AML`?